user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/TimeRangeFeed.md

```diff
@@ -22,6 +22,11 @@ To configure the component:
 
    - *Show refresh timer*: Determines whether an indication is displayed that the data will be refreshed.
 
+   - *Presets*: Available from DataMiner 10.3.10/10.4.0 onwards. Allows you to customize the presets displayed when selecting the component in read mode. The available presets for selection are: *Still busy*, *In the past*, *Recently*, *Long run*, *Starting from now*, *Near future*, and *Distant future*.
```

Comment:
Don't think we have to specify "in read mode", a user who is editing/configuring the timerange component can also open the overlay and see the presets.